src/sentry/services/hybrid_cloud/notifications/impl.py

```diff
@@ -171,6 +171,10 @@ def remove_notification_settings_for_user(
     def get_many(self, *, filter: NotificationSettingFilterArgs) -> List[RpcNotificationSetting]:
         return self._FQ.get_many(filter)
 
+    def remove_notification_settings_for_organization(self, *, organization_id: int) -> None:
+        assert organization_id
```

Comment:
```suggestion
        assert organization_id, "organization_id must be a positive value"
```

If this ever gets tripped a message in the captured issue will be helpful.